client/src/app/navbar/navbar-app/navbar-app.component.ts

```diff
@@ -19,14 +21,33 @@ export class NavbarAppComponent {
   mentees!: UserDTO[] | undefined;
   selectedMenteeName: string = 'Mentorship';
   user$: Observable<UserDTO | null | undefined> = this.fakeUserService.user$;
+  requests!: any[];
```

Comment:
nit: would be nice to have a more precise type than `any`: could it be something like `MentorshipRequestGetDTO[] | MentorshipResponseGetDTO[]`?